docs/configuration/examples.md

```diff
@@ -0,0 +1,104 @@
+# Configuration Examples
+
+## `ignore.yml`
+
+### General
+
+```yaml
+push:
+  # No need to sync build results to the remote machine.
+  # Replace "out" with the directory where build results are placed.
+  - "out"
+pull:
+  # No need to sync sources to the local machine.
+  # Replace "src" with the directory where the source code is placed.
+  - "src"
+```
+
+### Tools
+
+#### Git
+
+```yaml
+push:
+  # Synching ".git" or other VCS directories is a bad idea.
+  # They are usually heavy and are not required for the build.
+  - "/.git"
+```
+
+#### IntelliJ IDEA
+
+Includes Android Studio, PyCharm, WebStorm, GoLand and other tools built
+on the IntelliJ Platform.
+
+```yaml
+push:
+  # Synching IDE-specific directories is not required for the build.
+  - "/.idea"
```

Comment:
You can also add `*.iml`, because they're all over the place in IntelliJ projects